src/java.base/share/classes/java/util/zip/Inflater.java

```diff
@@ -636,14 +636,19 @@ public int getAdler() {
     }
 
     /**
-     * Returns the total number of compressed bytes input so far.
+     * Returns the total number of compressed bytes input so far, discarding
+     * the 32 highest order bits, as if the {@link #getBytesRead()}
+     * method was called followed by a narrowing conversion from <code>long</code>
+     * to <code>int</code>.
      *
-     * <p>Since the number of bytes may be greater than
-     * Integer.MAX_VALUE, the {@link #getBytesRead()} method is now
-     * the preferred means of obtaining this information.</p>
+     * @deprecated This method cannot safely return a result without a potential
```

Comment:
Perhaps word this as:
>
>This method cannot return the right value when the number of compressed bytes is greater than {@link Integer.MAX_VALUE}. It is recommended that the {@link #getBytesRead()} method be used instead.

Reply:
I've simplified the `@deprecated` note to the following:

> This method cannot return the correct value when the number of compressed bytes is 
> larger than {@link Integer#MAX_VALUE}. Use {@link #getBytesRead()} instead.